### PR TITLE
Scaled component

### PR DESCRIPTION
**Pull request by JonBeyond**
Completed functionality for:
1. Navigating left/right on the scaled photo
2. Clicking the favorite button 'favorites' the product (NOTE: will need to add a database entry for this.  Oops!)
3. Clicking on a thumbnail now shows the full sized image

### CODE REVIEW

File: client/src/components/carousel.jsx [hunk 1]
```diff
@@ -3,27 +3,75 @@ import Scaled from './scaled.jsx';
...
+    this.setState({
+      isFavorite: !this.state.isFavorite
+    });
+    //TODO: need to visually show the change, for now just log
+    console.log(`Is this product favorited? ${this.state.isFavorite ? 'yes' : 'no'}`);
+  }
+
+  handleThumbnailClick(val) {
+    this.setState({
+      scaled: val
```
Comment (jasonto510) [inline client/src/components/carousel.jsx]: Where is this val?

Comment (jasonto510) [reply client/src/components/carousel.jsx]: Oh, I'm guessing it increments every time it's clicked

File: client/src/components/scaled.jsx [hunk 2]
```diff
@@ -2,8 +2,13 @@ import styles from './styles.css.js';
 
 const Scaled = (props) => {
   return (
-    <div><img src={props.image} style={styles.scaledImage}></img></div>
+    <div>
+      <img src={props.image} style={styles.scaledImage}></img>
```
Comment (jasonto510) [inline client/src/components/scaled.jsx]: I like how you imported the stylesheet

File: client/src/components/carousel.jsx (hunk 1 above)
Comment (JonBeyond) [reply client/src/components/carousel.jsx]: It's the photo index that was clicked it, it comes from `thumbnail.jsx`, which is receives from the `map` function in `thumbnails.jsx`

**Pull request merged**